front/lib/resources/storage/models/data_source_view.ts

```diff
@@ -90,3 +102,9 @@ DataSourceViewModel.belongsTo(DataSource, {
   as: "dataSourceForView",
   foreignKey: { name: "dataSourceId", allowNull: false },
 });
+
+DataSourceViewModel.belongsTo(User, {
+  as: "editedByUser",
```

Comment:
Does this work? IIRC `Sequelize` prevents using twice the same name for relations. Given that this one is already used by `DataSourceModel` ...

Reply:
editedByUser is on dataSourceView side, not user - the "as" in only an alias replacing "user" by "editedByUser" in where and include clauses.
And yes, it works ;-) At least it does not crash and the user are correctly retrieved :-)